### PR TITLE
SK-197 Implemented search for conversations by name

### PR DESCRIPTION
**Pull request by Oleksandr1414**

### CODE REVIEW

**Review by ccvlad — APPROVED** [review]
LGFM

File: app/providers/operations/conversation/search/index.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+class ConversationSearchOperation {
+  constructor(conversationService) {
+    this.conversationService = conversationService
+  }
+
+  async perform(ws, searchParams) {
+    const ignoreIds = []
+
+    const limit =
```
Comment (IgorKhomenko) [inline app/providers/operations/conversation/search/index.js:11]: could be improved like this, w/o if/else 

```
const limit = Math.min(searchParams.limit || MAIN_CONSTANTS.LIMIT_MAX, MAIN_CONSTANTS.LIMIT_MAX);

```

**Comment by IgorKhomenko** [comment]
@Oleksandr1414 please do not forget to add tests

**Review by IgorKhomenko — CHANGES_REQUESTED** [review]
please do not forget to add tests

**Comment by IgorKhomenko** [comment]
@Oleksandr1414 please also cover documentation for new API

**Review by IgorKhomenko — APPROVED** [review]

**Pull request merged**